__tests__/endpoints.test.js

```diff
@@ -0,0 +1,283 @@
+const request = require('supertest')
+const app = require('../app.js')
+const db = require('../db/data/test-data/index.js')
+const seed = require('../db/seeds/seed.js')
+
+beforeEach(()=> seed(db))
+
+describe('Topic Tests', () => {
+    test('Status 200: GETs all Topics',()=>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body })=>{
+            const topics = body
+            expect(topics).toHaveLength(3)
+            topics.forEach((topic)=> {
+                expect(topic).toMatchObject({
+                    description : expect.any(String),
+                    slug : expect.any(String),
+            })
+
+            })
+        })
+    })
+})
+
+describe('Endpoint Tests', () => {
+    test('Status 200: GETs all EndPoints Info',()=>{
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then((data)=>{
+            const endpoints = data.body
+            
+            for (let endpoint in endpoints) {
+             const endpointObject = endpoints[endpoint]
+             const length = Object.keys(endpointObject).length
+
+             switch(length) {
```

Comment:
Nothing to comment on really, just wanted to say I'm quite excited to see a switch statement - I rarely get to see these out in the wild 😂